README.md

```diff
@@ -30,6 +30,16 @@ to avoid duplicates.
 Contributions are always welcome. The preferred way to receive code contribution is by submitting a 
 Pull Request on github.
 
+## Developing
+
```

Comment:
````suggestion

1. Clone the [ArduinoCore-API](https://github.com/arduino/ArduinoCore-API) repo to any convenient location:
   ```
   git clone https://github.com/arduino/ArduinoCore-API.git
   ```
````

I think this is a significant enough operation that it is worth a dedicated step rather than being mentioned only as an afterthought in the symlink step.

Here I have proposed the use of fenced code blocks rather than inline markup for the commands. I think this improves readability by avoiding wrapping and also makes it easy to copy the commands by clicking the icon GitHub generates for these blocks.